scipy/stats/_stats_py.py

```diff
@@ -292,7 +292,7 @@ def gmean(a, axis=0, dtype=None, weights=None):
     return np.exp(np.average(log_a, axis=axis, weights=weights))
 
 
-def hmean(a, axis=0, dtype=None):
+def hmean(a, axis=0, dtype=None, weights=None):
```

Comment:
Corrected.

Reply:
A minor thing to consider 😃 Please use the suggestions next time. It's both convenient (and you can group all suggestions in a single commit: meaning if you put a good message and respect our naming conventions with the first 3 letters code, we could keep the history instead of squashing the PR) for you and gives credit to the reviewers (marked as co-author then).

Reply:
You are right: I will do this next time!